src/Microsoft.IdentityModel.Tokens/Validation/Results/Details/IssuerValidationError.cs

```diff
@@ -13,19 +13,10 @@ internal IssuerValidationError(
             MessageDetail messageDetail,
             Type exceptionType,
             StackFrame stackFrame,
-            string? invalidIssuer)
-            : this(messageDetail, ValidationFailureType.IssuerValidationFailed, exceptionType, stackFrame, invalidIssuer, null)
-        {
-        }
-
-        internal IssuerValidationError(
-            MessageDetail messageDetail,
-            ValidationFailureType validationFailureType,
-            Type exceptionType,
-            StackFrame stackFrame,
             string? invalidIssuer,
-            Exception? innerException)
-            : base(messageDetail, validationFailureType, exceptionType, stackFrame, innerException)
+            ValidationFailureType? validationFailureType = null,
```

Comment:
Be consistent with the naming.
Here we have validationFailureType, above in AudienceValidationError  the parameter is failurType.
I suggest validationFailureType.